### DISCUSSION TITLE
multi-artist album/playlist split by artist

### DISCUSSION THREAD

**Discussion by anarcat** [navidrome/navidrome#2668] [Q&A] [answered]
Hi!

I'm trying to figure out how to curate this particular folder in my collection. It's a copy of a Youtube playlist I have downloaded with yt-dlp (i know, not the cleanest, but this playlist is important to me), so the metadata was built up by hand.

I've tried various things to make this show up properly in Navidrome. The "album" tag is all the same on all tracks, and I have set the "albumartist" tag to "Various Artists", and the "compilation" tag is set to `1` (as per [this FAQ](https://www.navidrome.org/docs/faq/#-i-have-an-album-with-tracks-by-different-artists-why-is-it-broken-up-into-lots-of-separate-albums-each-with-their-own-artist)). The "artist" tag points to the "right" artist that varies according to the track (because I would still like the metadata).

Unfortunately, my "recently added" now looks like this:

![recently added tab showing all "blue" albums with the same album/artist name](https://github.com/navidrome/navidrome/assets/796623/810d296b-7548-46c7-abf0-5a9efd072a90)

Here's a metadata dump for the first two tracks:

```
anarcat@angela:play liste de ma mort (mel)$ ffprobe 001* 2>&1| sed '0,/.*Metadata:/d' 
      artist          : Kumbia Queers
      artists         : Kumbia Queers
      encoder         : Lavf59.27.100
      originaldate    : 2019-01-07
      date            : 2019-01-07
      title           : La despedida
      script          : Latn
      originalyear    : 2019
      album           : Playlist de ma mort (mel)
      compilation     : 1
      album_artist    : Various Artists
anarcat@angela:play liste de ma mort (mel)$ ffprobe 002* 2>&1| sed '0,/.*Metadata:/d' 
      artist          : Moby
      label           : Arcade
      artists         : Moby
      encoder         : Lavf59.27.100
      originaldate    : 2001-08-06
      date            : 2001-08-06
      title           : Porcelain
      script          : Latn
      isrc            : GB5WK1499388;GBAJH9900036
      originalyear    : 2001
      album           : Playlist de ma mort (mel)
      compilation     : 1
      album_artist    : Various Artists
```

It seems to me navidrome should be regrouping those, no? What am I doing wrong?

**Comment by metalheim** [comment, marked as answer]
My best guess is that the album is Split by year, or origyear (not sure which must be the same so that theyre still grouped)

Maybe setting `Scanner.GroupAlbumReleases` to `true` helps in this case? https://www.navidrome.org/docs/usage/configuration-options/#available-options

**Reply by anarcat** [reply]
that was it! tweaking the `Scanner.GroupAlbumReleases` did *not* actually work for me, but I removed dates on the actual files as I didn't really need that deep metadata for those files, and that merged the album...

update: sent a PR on the website to improve docs: https://github.com/navidrome/website/pull/142

phew, thanks!
